ci/travis_pypi_setup.py

```diff
@@ -23,10 +22,12 @@
 
 GITHUB_REPO = 'plus3it/watchmaker'
 TRAVIS_CONFIG_FILE = os.path.join(
```

Comment:
Why not remove all of this? Where is it being called? I think you removed all uses of it, remove 24 - 30 as well.